JM/CF/Scripts/3_Game/CommunityFramework/Expression/CF_MathExpression.c

```diff
@@ -0,0 +1,153 @@
+class CF_MathExpression : CF_Expression
+{
+	//! I want this to be protected or private but a compile error happens
```

Comment:
When talking about I, maybe put a `Jacob:` in front so others know who to talk to if they want info on this.